packages/ui/src/common/components/selects/components.tsx

```diff
@@ -24,9 +24,11 @@ interface Props {
   disabled?: boolean
 }
 
-export const Option = ({ children, onClick, disabled }: Props) => (
+export const Option = ({ children, onClick, disabled, ...props }: Props) => (
   <OptionComponentContainer onClick={onClick} disabled={disabled}>
-    <OptionComponent disabled={disabled}>{children}</OptionComponent>
+    <OptionComponent disabled={disabled} {...props}>
```

Comment:
@gyroflaw
> All these changes are required due to styling. We use `Styled-component`(below library) for styling and [here](https://github.com/gyroflaw/pioneer/blob/506edb8c4e346fb9d2cc477e19de7f5a22173b9e/packages/ui/src/working-groups/components/SelectWorkingGroupOpening/OptionsListWorkingGroupOpening.tsx#L40) styles `Option` component
> 
> ```ts
> const StyledOption = styled(Option)`
> ```
> 
> and the library generates a CSS class for that style and assigns it to the component `className` props. If we omitted `{...props}` in the component this style won't be applied.
> 
> ![image](https://user-images.githubusercontent.com/83718263/211153597-67feddf7-894b-4077-8213-e4cf39f1df8c.png)

Right I missed that for `Option`, however only the `className` prop is needed to get the styled method to work (see: https://styled-components.com/docs/basics#styling-any-component).

It would better to just pass `className` because it makes it clear that the style is what is being passed:
```ts
export const Option = ({ children, onClick, disabled, className }: Props) => (
  <OptionComponentContainer onClick={onClick} disabled={disabled}>
    <OptionComponent disabled={disabled} className={className}>{children}</OptionComponent>
```